### PR TITLE
US 01.05.01 Create view for saved ingredients #7

### LINKED ISSUE #7: US 01.05.01 Create view for saved ingredients
As a meal planner, I want to see a list of my ingredients.

### PR DESCRIPTION
**Pull request by kstark1**
Included basic functionality for viewing ingredients list. Does not have interaction with the database yet.

### CODE REVIEW

**Comment by sruduke** [comment]
Approved

File: app/src/main/java/com/example/happymeals/AmountUnit.java [hunk 1]
```diff
@@ -0,0 +1,5 @@
+package com.example.happymeals;
+
+public enum AmountUnit {
```
Comment (jeastgaard) [inline app/src/main/java/com/example/happymeals/AmountUnit.java]: Why do we have an entire file dedicated to an enum?

File: app/src/main/java/com/example/happymeals/Ingredient.java [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+ */
+public class Ingredient {
+    private String description;
+    private String bestBeforeDate;
+    private Location location;
+    private int amount;
+    private AmountUnit unit;
+    private IngredientCategory category;
+
+    public Ingredient(String description, String bestBeforeDate, Location location, Integer amount, AmountUnit unit, IngredientCategory category) {
```
Comment (jeastgaard) [inline app/src/main/java/com/example/happymeals/Ingredient.java]: Need an empty constructor as well for database serialization.

File: app/src/main/java/com/example/happymeals/IngredientCategory.java [hunk 3]
```diff
@@ -0,0 +1,5 @@
+package com.example.happymeals;
+
+public enum IngredientCategory {
```
Comment (jeastgaard) [inline app/src/main/java/com/example/happymeals/IngredientCategory.java]: Enum file? Should we not be putting everything into a constants file?

File: app/src/main/java/com/example/happymeals/Location.java [hunk 4]
```diff
@@ -0,0 +1,5 @@
+package com.example.happymeals;
+
+public enum Location {
```
Comment (jeastgaard) [inline app/src/main/java/com/example/happymeals/Location.java]: Enum File?

File: app/src/main/java/com/example/happymeals/MainActivity.java [hunk 5]
```diff
@@ -10,5 +11,7 @@ public class MainActivity extends AppCompatActivity {
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_main);
+        Intent intent = new Intent(getApplicationContext(), IngredientStorageActivity.class);
```
Comment (jeastgaard) [inline app/src/main/java/com/example/happymeals/MainActivity.java]: Debug code should be removed for pull.

**Review by jeastgaard — CHANGES_REQUESTED** [review]
We should not have files dedicated to small enumerations. A single constants file should be more than sufficient for this project.

**Review by jeastgaard — APPROVED** [review]

**Pull request merged**